beetsplug/check.py

```diff
@@ -139,21 +143,61 @@ def copy_original_checksum(self, config, task):
 
     def verify_import_integrity(self, session, task):
         integrity_errors = []
+        failed_items = []
         if not task.items:
             return
         for item in task.items:
             try:
                 verify_integrity(item)
             except IntegrityError as ex:
                 integrity_errors.append(ex)
+                failed_items.append(item)
```

Comment:
Let’s have one list for both: `failed_items.append((ex, item))`. This will make the handling code flow a bit nicer